dev/Repeater/ItemsRepeater.idl

```diff
@@ -334,12 +334,14 @@ unsealed runtimeclass ElementAnimator
     protected void OnBoundsChangeAnimationCompleted(Windows.UI.Xaml.UIElement element);
 }
 
-[WUXC_VERSION_PREVIEW]
+[WUXC_VERSION_MUXONLY]
 [webhosthidden]
 runtimeclass ScrollAnchorProvider : Windows.UI.Xaml.Controls.ContentControl
 {
     ScrollAnchorProvider();
 
+    Windows.UI.Xaml.UIElement AnchorElement{ get; };
```

Comment:
The property on our existing IScrollAnchorProvider is "CurrentAnchor" rather than AnchorElement.

Reply:
I can update the naming once the API review is done.